shared/actions/login/index.js

```diff
@@ -91,47 +91,54 @@ function getAccounts (): AsyncAction {
 
 export function login (): AsyncAction {
   return (dispatch, getState) => {
-    const deviceType = isMobile ? 'mobile' : 'desktop'
-    const incomingMap = makeKex2IncomingMap(dispatch, getState)
-    const params : login_login_rpc = {
-      ...makeWaitingHandler(dispatch),
-      method: 'login.login',
-      param: {
-        deviceType,
-        usernameOrEmail: '',
-        clientType: enums.login.ClientType.gui
-      },
-      incomingCallMap: incomingMap,
-      callback: (error, response) => {
-        if (error) {
-          dispatch({
-            type: Constants.loginDone,
-            error: true,
-            payload: error
-          })
-
-          if (!isRPCCancelError(error)) {
-            dispatch(routeAppend({
-              parseRoute: {componentAtTop: {component: Error, props: {
-                error,
-                onBack: () => dispatch(cancelLogin())
-              }}}
-            }))
+    const props = {
```

Comment:
we always ask for username first. the login rpc takes an optional username but this uses the last user, which we don't want to happen in relogin if they chose 'someone else' as the option